ui/perfherder/compare/CompareTable.jsx

```diff
@@ -79,6 +91,58 @@ export default class CompareTable extends React.PureComponent {
     }
   };
 
+  // Sorts the specified key of the array data depending on the typeof key
+  // If something changes in columns, you just need to update sortableKeys
+  // with the corresponding type
+  // The data model of the sortable keys is:
+  // name: string
+  // originalValue: number
+  // newValue: number
+  // deltaPercentage: number
+  // confidence: number
+  // originalRuns: array
+  // newRuns: array
+  // default: string
```

Comment:
This whole chunk is redundant since it's all declared in `sortableKeys` :)

Reply:
Sorry, I put it there because this whole sort thing seemed a bit complicated and I wanted to help a bit with comments.
Also, I wouldn't call an object using a verb (I called the function `sortBy`). :) Considering the purpose of this object (line 126 & 130), probably `byKey` would be enough descriptive. Hope you find it proper.